### PR TITLE
chore: update to 4.32

### PR DESCRIPTION
**Pull request by andygup**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,161 +1,162 @@
...
+-	ValidateNetworkTopologyResult.dirtyAreaCount deprecated since version 4.28. Dirty area count was implemented in the original version of utility network, but as of schema version 4 of the utility network, this is no longer supported.
+-	ValuePicker.own deprecated since 4.28 Use addHandles() instead.
+-	VersionManagementViewModel.versionIdentifierLookup deprecated since version 4.30. Use VersioningState instead.
+-	VersionManagementViewModel.versionInfoLookup deprecated since version 4.30. Use VersioningState instead.
+-	VersionManagementViewModel.versionManagementServiceLookup deprecated since version 4.30. Use VersioningState instead.
+-	VideoPlayer.own deprecated since 4.28 Use addHandles() instead.
+-	WCSLayer.rasterInfo deprecated since 4.29. Use serviceRasterInfo instead.
+-	Weather.own deprecated since 4.28 Use addHandles() instead.
+-	Widget.own deprecated since 4.28 Use addHandles() instead.
+-	Zoom.own deprecated since 4.28 Use addHandles() instead.
```
Comment (annelfitz) [inline CHANGELOG.md:160]: the formatting is a little funky on these. maybe remove the extra spaces?

**Review by annelfitz — APPROVED** [review]
lgtm!

File: CHANGELOG.md (hunk 1 above)
Comment (andygup) [reply CHANGELOG.md:160]: Oh that's odd. I'll see what I can do...I don't see that spacing in my IDE.

image

Comment (andygup) [reply CHANGELOG.md:160]: Maybe it's a github thing. In the page preview it looks okay?

image

Comment (andygup) [reply CHANGELOG.md]: nm, I figured it out. I'm fixing it now.

Comment (andygup) [reply CHANGELOG.md]: Fixed!

Comment (annelfitz) [reply CHANGELOG.md]: nice, looks good!

**Pull request merged**